<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NLSimpleGuiWindow</class>
 <widget class="QMainWindow" name="NLSimpleGuiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NLSimpleGuiWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>200</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>353</width>
            <height>75</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="TQtWidget" name="m_mathFormulaWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>100</horstretch>
               <verstretch>25</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="geneticOptimizeButton">
            <property name="text">
             <string>Genetic Optimize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="baysianFineTuneButton">
            <property name="text">
             <string>Baysian Fine Tune</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addCgmsButton">
            <property name="text">
             <string>Add CGMS Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addMealDataButton">
            <property name="text">
             <string>Add Meal Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addMeterDataButton">
            <property name="text">
             <string>Add Meter Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addCustonDataBustom">
            <property name="text">
             <string>Add Custom Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addExcersizeButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Add Excersize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="redrawButton">
            <property name="text">
             <string>Redraw</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="modelDisplyTab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="m_modelDisplayScrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>732</width>
                 <height>434</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="TQtWidget" name="m_modelDisplayWidget" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="optionsTab">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="clarkeGridTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Page</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="clarkLeftVLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Clarke Error Grid Analysis:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="TQtWidget" name="clarkeLegendWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>165</width>
                  <height>185</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="TQtWidget" name="clarkResultsWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>165</width>
                  <height>86</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="clarkButtonGroupBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>Grid Type</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="clarkCgmsVMeterRadioButton">
                   <property name="text">
                    <string>CGMS V Meter</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="clarkePredVCGMSRadioButton">
                   <property name="text">
                    <string>Pred V CGMS</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="clarkeRefreshButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="TQtWidget" name="clarkeErrorGridWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TQtWidget</class>
   <extends>QWidget</extends>
   <header>TQtWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>setupForRecentlyDefinedModel()</slot>
 </slots>
</ui>
